extensions/amp-story/amp-story.md

```diff
@@ -362,6 +374,48 @@ Usage: `<amp-story ... supports-landscape>...</amp-story>`
   </amp-anim>
 </figure>
 
+### Live story
+
+Using the `live-story` attribute, you can append new pages to a story for users to see in real-time. If the user is on the last page it will show a notification saying there are new pages and the progress bar will be updated.
+
+The core use cases for this are: coverage for breaking news or live events where the user can stay on to see new pages as they come in. Common examples are award shows, sporting events, and elections.
+
+#### How it works
+
+In the background, while an AMP Story using `live-story` is displayed on the client, the AMP runtime polls the origin document on the host for updates. When the client receives a response, it then filters and dynamically inserts those updates back into the story on the client. Publishers can customize the polling rate in order to control the number of incoming requests, and AMP caches like the Google AMP Cache can perform optimizations to reduce the server response payload, saving client bandwidth and CPU cycles.
+
+#### Polling
+
+In most implementations for live blogs, content is either pushed by the server to the client instance of a page, or the client polls a JSON endpoint to receive updates. The implementation here is different, in that the client instance of the story polls the server copy of the story document for updates inside the `<amp-story>` element. For instance: if the user is viewing a story served from an AMP cache, the client will poll that document hosted on that AMP cache for updates; if the user is viewing a document served from a web publisher's origin domain (e.g. "example.com"), then the client will poll the document hosted on that origin domain for updates.
+
+This means that publishers of stories do not need to set up a JSON endpoint or push mechanism for this functionality to work. New content just needs to be published to the same URL, with a valid amp-story markup, and the user will have that content pulled into their client instance during the next poll (poll intervals are configurable in the component, and are valid above the minimum of 15 seconds).
```

Comment:
Second sentence is a bit long, consider breaking it up. Since it is required for developers to specify the time in milliseconds we should update the language to reflect that instead of saying `15 seconds`.